lib/options.js

```diff
@@ -108,6 +108,12 @@ module.exports = optionator({
         description: "Report errors only"
     },
     {
+        option: "max-warnings",
+        type: "Number",
+        default: "-1",
+        description: "Maximum warning count before eslint exits with an error (or -1 for no limit)"
```

Comment:
On it. Is the longer description still okay in the Markdown file, or do those need to be absolutely in sync with the short descriptions in `lib/options.js`?